src/components/interpolatedColorLegend.ts

```diff
@@ -68,7 +68,7 @@ export module Components {
      */
     public formatter(formatter: Formatter): InterpolatedColorLegend;
     public formatter(formatter?: Formatter): any {
-      if (formatter === undefined) {
+      if (formatter === null) {
```

Comment:
Can you explain the bug?  The way that you have currently implemented this change completely changes the functionality of this function.  Essentially, I believe the getter is now called with `icl.formater(null);` which seems very undesirable.